src/main/resources/assets/paths/daaRequestByDaaId.yaml

```diff
@@ -0,0 +1,20 @@
+post:
+  summary: Request that a Signing Official create a DAA-Library Card Relationship
+  operationId: Request a DAA-Library Card Relationship
```

Comment:
Can you help direct me to the definition of this operation in your PR? I'm having a hard time finding it.

(I am also used to seeing `operationIds` matching up to the method name of an API controller, so no spaces and not necessarily matching the description.)

Reply:
I think we've not had a solid understanding of `operationId` at all. They will need to be unique, which can be hard to guarantee if we have similar API methods that do similar things. I suggest we remove them since they are optional
https://swagger.io/docs/specification/paths-and-operations/